tokio-threadpool/src/lib.rs

```diff
@@ -3,6 +3,107 @@
 #![doc(html_root_url = "https://docs.rs/tokio-threadpool/0.1.4")]
 #![deny(warnings, missing_docs, missing_debug_implementations)]
 
+// The Tokio thread pool is a thread pool designed to scheduled futures in Tokio
+// based applications. The thread pool structure manages two sets of threads:
+//
+// * Worker threads.
+// * Backup threads.
+//
+// Worker threads are primary threads and are used to schedule futures. These
```

Comment:
is a "primary thread" something i should know about? Otherwise, this might be clearer as:

"The primary thread pool consists of worker threads and is used to schedule futures using a work-stealing strategy. Back up threads, on the other hand, are intended only to support the `blocking` API. Threads may transition between these pools.